WordPress/src/main/java/org/wordpress/android/ui/mysite/ScanAndBackupSource.kt

```diff
@@ -15,17 +16,19 @@ class ScanAndBackupSource
     private val jetpackCapabilitiesUseCase: JetpackCapabilitiesUseCase
 ) : MySiteSource<JetpackCapabilities> {
     override fun buildSource(siteId: Int) = flow {
-        emit(JetpackCapabilities(scanAvailable = false, backupAvailable = false))
```

Comment:
We don't need to always set visibility to false before the request to fetch capabilities finished, since the use case now always returns the cached value first.